package.json

```diff
@@ -11,7 +12,10 @@
         "date-fns": "^2.28.0",
         "formik": "^2.2.9",
         "ky": "^0.31.0",
+        "moment": "^2.29.4",
         "react": "^18.2.0",
+        "react-circular-progressbar": "^2.1.0",
+        "react-datepicker": "^4.8.0",
```

Comment:
Was scrolling through the Chakra UI docs and found a browser-native date-picker using their `Input` component:
https://chakra-ui.com/docs/components/input#input-methods-other-than-text